components/widgets/icons/fa_next_icon.tsx

```diff
@@ -1,21 +1,20 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 
 import {FormattedMessage} from 'react-intl';
 
-export default class NextIcon extends React.PureComponent {
-    static propTypes = {
-        additionalClassName: PropTypes.string,
-    };
+type Props = {
+    additionalClassName: string | null;
+}
 
-    static defaultProps = {
+export default class NextIcon extends React.PureComponent<Props> {
+    public static defaultProps: Props = {
         additionalClassName: null,
     };
 
-    render() {
+    public render() {
```

Comment:
@devinbinnie This is fine but we're not consistent.  In other components, we're explicitly assigning `JSX.Element`

Reply:
Yes, we should be specifying return types everywhere.